File: slides/DDD-Prevalence/PrevalenceJava/src/breizhcamp/demo/Personne.java
package breizhcamp.demo;

import java.lang.String;
import java.io.Serializable;

public class Personne implements Serializable {

	private static final long serialVersionUID = 920394837439199618L;

	private String _Prenom;
	private String _Nom;
	private int _Age;
	
	Personne(String Prenom, String Nom, int Age) {
		_Prenom = Prenom;
		_Nom = Nom;
		_Age = Age;
	}
}

File: slides/DDD-Prevalence/PrevalenceCSharp/BambooBreizhCamp/Personne.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace BambooBreizhCamp
{
    [Serializable]
    class Personne
    {
        public string Nom { get; set; }
        public string Prenom { get; set; }
        public int Age { get; set; }
    }
}
